.flake8

```diff
@@ -0,0 +1,7 @@
+[flake8]
+builtins =
+	_
+	_n
```

Comment:
Ooh, I missed the _n when I added `builtins = _` to setup.cfg!

Speaking of which, we should merge the two flake8 config sections into one file.  Whichever you prefer.

Reply:
Fewer files is better. Not sure of the balance between a hidden file that is called flake8 and a visible file that is called something else.